administration/package.json

```diff
@@ -26,6 +26,7 @@
     "react-dom": "^18.2.0",
     "react-flip-move": "^3.0.5",
     "react-router-dom": "^6.14.0",
+    "react-to-print": "^2.14.13",
```

Comment:
couldn't find where this is in use? Can be removed?

Reply:
Jap. Missed that 👍🏼